### PR TITLE
Release 0.0.6

### PR DESCRIPTION
**Pull request by kpp**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+# 0.0.6 (August 27, 2018)
+
+* (#)
```
Comment (kurnevsky) [inline CHANGELOG.md]: What does it stand for?

Comment (kpp) [reply CHANGELOG.md]: It's a WIP. I will add more PRs (if they will be merged) until August 27.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+* Relicense to MIT/GPLv3+ (#185)
+* Fix hole punching (#183, #193)
+* Simplify example and synchronize it with tox-node (#182)
+* Use DhtNode instead of PackedNode (#181)
+* Update the status of the project (#180)
+* Send periodic pings in TCP server (#178)
+* Fix binary search in Kbucket (#177)
+* Add crypto_core::random_usize (#176)
+* Add mutable iterator for Kbucket (#175)
+* DHT refactoring, NAT fixes (#174, #179, #193) // FIXME
```
Comment (kurnevsky) [inline CHANGELOG.md]: FIXME?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+* Parse packet type FriendRequests (#211)
+* Add parameter for eviction strategy (#210)
+* Speed up 'cargo test' for tcp (#209)
+* Simplify constructor of Kbucket (#207)
+* Refactor SockAndTime struct (#206)
+* Parse packet type ShareRelays (#205)
+* Add fake friends to DHT (#204)
+* Handle all connections in ServerExt::run (#203)
+* Renaming and minor refactoring (#202)
+* Add packet parser for ID_ALIVE (#201, #208)
```
Comment (NamsooCho) [inline CHANGELOG.md]: How about Alive instead of ID_ALIVE

Comment (kpp) [reply CHANGELOG.md]: Sure!

File: CHANGELOG.md (hunk 2 above)
Comment (kpp) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (kpp) [reply CHANGELOG.md]: done

**Pull request merged**